code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -67,6 +67,8 @@
 
 /obj/machinery/atmospherics/examine(mob/user)
 	. = ..()
+	. += "<span class='notice'>[src] is on layer [piping_layer].</span>"
+	. += "<span class='notice'>[src] is of [GLOB.pipe_color_name[pipe_color]] color"
```

Comment:
How will this impact atmos machinery that doesn't have meaningful values for these things?

Reply:
There is no atmos machine that doesn't have these two values since they are set directly here as base (layer 3 and color grey). 
So all components will just show the default examine